recipes/reprit/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "reprit" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1e5198c852589518814118fa72dfd0b975e6633fed4c41ec22d69e1824b3b458
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - {{ name }}
```

Comment:
Let's avoid jinja template variables here and in the `about` section. The same for the other recipe. It's not incorrect, but stylistically, we tend to only use them under the `package` and `source` section and possibly in the `doc_url` for the version (but most recipes don't).

Reply:
Done